.github/workflows/changelog.yml

```diff
@@ -0,0 +1,36 @@
+name: Changelog
+# When a new tag pushed
+# - Generate changelogs from formatted commit messages
+# - Update the latest release body with generated changelogs
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Almost there, you miss the `checkout` step.

```
    - name: Checkout
      uses: actions/checkout@v2
      with:
        fetch-depth: 0
```

Reply:
Thank you! I added checkout in `changelog.yml`. We need the full history to generate changelog but not for release, right?